less-print/viewdetails.less

```diff
@@ -1,5 +1,90 @@
 .view-details-associations, .view-details-comments, .view-details-photos,
-.scroll-top-btn, .view-details-map, .location-static, .badge, .icons-header,
-.logo {
-  display: none;
+.scroll-top-btn, .view-details-map, .badge, .icons-header {
+  display: none !important;
+}
+
+.view-details-title{
+	display: block;
+	font-size: .75em !important;
+	margin-top: 0px;
+	margin-bottom: 10px;
+}
+
+.description , .view-details-info {
+	display: block !important;
```

Comment:
There is a bug with Firefox, with long routes (such as https://www.camptocamp.org/routes/53940/fr/mont-aiguille-la-tour-des-gemeaux) : the Matériel and extras infos parts are cutting the Description part in two : 
![image](https://cloud.githubusercontent.com/assets/24532066/21220249/4fbafa38-c2b8-11e6-99e3-591f95297eb2.png)

Yesterday, seems to me that the .description to display: block was solutionning the problem, but no. So there is no reason to have .description to display: block.
I'm working on that.